vql/windows/filesystems/ntfs_nonwindows.go

```diff
@@ -0,0 +1,30 @@
+//go:build !windows
+// +build !windows
+
+package filesystems
```

Comment:
This is great to make the accessor work on linux but it has to be done in a way that we can still do the same things on linux or windows - so allow a windows binary to also deal with images.

Ultimately this will be a specialised accessor that remaps a directory based on the remapping configuration.